docs/security/trust/trust_sandbox.md

```diff
@@ -61,146 +63,114 @@ work exclusively with the Docker Hub, you would not need with these components.
 They are built into the Docker Hub for you. For the sandbox, however, you must
 build your own entire, mock production environment and registry.
 
-### Configure /etc/hosts
-
-The sandbox' `notaryserver` and `sandboxregistry` run on your local server. The
-client inside the `notarysandbox` container connects to them over your network.
-So, you'll need an entry for both the servers in your local `/etc/hosts` file.
-
-1. Add an entry for the `notaryserver` to `/etc/hosts`.
-
-        $ sudo sh -c 'echo "127.0.0.1 notaryserver" >> /etc/hosts'
-
-2. Add an entry for the `sandboxregistry` to `/etc/hosts`.
-
-        $ sudo sh -c 'echo "127.0.0.1 sandboxregistry" >> /etc/hosts'
-
-
 ### Build the notarytest image
 
-1. Create a `notarytest` directory on your system.
+1. Create a `trustsandbox` directory on your system and change into it.
 
-        $ mkdir notarysandbox
+        $ mkdir trustsandbox
+        $ cd trustsandbox
 
-2. Change into your `notarysandbox` directory.
+2. Create a filed called `Dockerfile` with your favorite editor.  For example, using vim:
```

Comment:
I think we should just prepackage this as an image, have all the instructions simple do `docker run docker/trustsandbox` and have an appendix of how to build your own trustsandbox image.